app/src/main/kotlin/com/hippo/ehviewer/ui/scene/GalleryDetailFragment.kt

```diff
@@ -942,10 +943,19 @@ fun GalleryDetailScreen(args: GalleryDetailScreenArgs, navigator: DestinationsNa
             lub.keyword = uploader
             navigator.navigate(GalleryListScreenDestination(lub))
         }
+
+        var showBottomSheet by remember { mutableStateOf(false) }
+
+        if (showBottomSheet && galleryDetail != null) {
+            ModalBottomSheet(onDismissRequest = { showBottomSheet = false }) {
+                GalleryInfoBottomSheet(galleryDetail, navigator)
+            }
```

Comment:
https://issuetracker.google.com/issues/278679016
https://issuetracker.google.com/issues/281967264

Reply:
> https://issuetracker.google.com/issues/278679016 https://issuetracker.google.com/issues/281967264

Got it, will grab BottomSheet in next CL